### ISSUE TITLE
Component Manager setup/pub/angular/angular.js Error

### ISSUE THREAD

**Issue by techgh0st** [magento/magento2#6203]
### Preconditions
1. Magento 2.1.0 with Sample Data
2. PHP 5.6.20
3. Mysql : Server version: 5.6.28-76.1 Percona Server (GPL), Release 76.1, Revision 5759e76
4. Apache/2.2.31 (Unix)
5. CentOS release 6.7 (Final)
### Steps to reproduce
1. Fresh install with sample data.
2. Log into admin dashboard.
3. Go to System -> Web Setup Wizard -> Component Manager

In the developer console, the following issue with the angular js script located at setup/pub/angular/angular.js is logged:

> The specified value "{{currentPage}}" is not a valid number. The value must match to the following regular expression: -?(\d+|\d+.\d+|.\d+)([eE][-+]?\d+)?

Here is the section of the script that is being noted by the developer console:

``` javascript
                while(index < length) {
                    if ( ((startIndex = text.indexOf(startSymbol, index)) != -1) &&
                        ((endIndex = text.indexOf(endSymbol, startIndex + startSymbolLength)) != -1) ) {
                        (index != startIndex) && parts.push(text.substring(index, startIndex));
                        parts.push(fn = $parse(exp = text.substring(startIndex + startSymbolLength, endIndex)));
                        fn.exp = exp;
                        index = endIndex + endSymbolLength;
                        hasInterpolation = true;
                    } else {
                        // we did not find anything, so we have to add the remainder to the parts array
                        (index != length) && parts.push(text.substring(index));
                        index = length;
                    }
                }
```

**Comment by andimov** [comment]
@techgh0st 
Thank you for reporting!
I've created an internal ticket  MAGETWO-57267 for this issue and linked it to this one.

**Comment by magento-engcom-team** [comment]
@techgh0st, thank you for your report.
The issue is already fixed in develop branch, 2.2.0

**Comment by roseofgold** [comment]
What commit corrects this issue in 2.1.12?
